src/Cms/Language.php

```diff
@@ -68,14 +68,15 @@ public function __construct(array $props)
 		$this->single       = $props['single'] ?? false;
 		$this->slugs        = $props['slugs'] ?? [];
 		$this->smartypants  = $props['smartypants'] ?? [];
-		$this->translations = $props['translations'] ?? [];
 		$this->url          = $props['url'] ?? null;
 
 		if ($locale = $props['locale'] ?? null) {
 			$this->locale = Locale::normalize($locale);
 		} else {
 			$this->locale = [LC_ALL => $this->code];
 		}
+
+		$this->variables = new LanguageVariables($this, $props['translations'] ?? []);
```

Comment:
I think we could already support a new variables prop here as well and fallback to translations to keep them available. 

```suggestion
		$this->variables = new LanguageVariables($this, $props['variables'] ?? $props['translations'] ?? []);
```